src/node_http2.cc

```diff
@@ -2456,7 +2457,7 @@ void Http2Session::Request(const FunctionCallbackInfo<Value>& args) {
   Environment* env = session->env();
 
   Local<Array> headers = args[0].As<Array>();
-  int32_t options = args[1]->Int32Value(env->context()).ToChecked();
+  int32_t options = args[1]->Int32Value(env->context()).FromMaybe(0);
```

Comment:
```suggestion
  int32_t options;
  if (!args[1]->Int32Value(env->context()).To(&options)) return;
```

Reply:
I'm just going to revert these back to ToChecked and change them in a separate PR.